packages/components/src/components/tabs-bar/tabs-bar-styles.ts

```diff
@@ -112,10 +114,10 @@ export const getComponentCss = (
       display: 'block',
       position: 'absolute',
       width: 0,
-      height: weight === 'semi-bold' ? '2px' : '1.5px',
+      height: isHighContrastMode ? '4px' : weight === 'semi-bold' ? '2px' : '1.5px',
       left: 0,
-      bottom: '-4px',
-      background: primaryColor,
+      bottom: isHighContrastMode ? '-1px' : '-4px',
+      background: isHighContrastMode ? canvasTextColor : primaryColor,
```

Comment:
Call of `getHighContrastColors()` without destructuring could go here since it is unnecessary for most every day use cases.